tensorflow/c/kernels/BUILD

```diff
@@ -58,12 +114,16 @@ tf_cc_test(
 filegroup(
     name = "android_all_op_kernels",
     srcs = [
-        "bitcast_op.cc",
+        "bitcast_op.cc", 
+        "summary_op.cc",
     ],
 )
 # LINT.ThenChange(//tensorflow/contrib/makefile/tf_op_files.txt)
 
 filegroup(
     name = "android_all_ops",
-    srcs = ["ops/bitcast.cc"],
+    srcs = [
+        "ops/bitcast.cc", 
```

Comment:
Remove the empty space at the end of this line.

Reply:
Awesome, thanks for the revisions, Gunan!